activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Deprecate implicit coercion of `ActiveSupport::Duration`
```

Comment:
This changelog is confusion. What are the cases where an implicit coercion is made?